Steps4Impact.xcodeproj/project.pbxproj

```diff
@@ -27,7 +27,6 @@
 		260BF5B923250414005F0882 /* SettingsTitleCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 260BF5B823250414005F0882 /* SettingsTitleCell.swift */; };
 		2611A4EF2335F8350030E77E /* Steps4ImpactSpec.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2611A4EE2335F8350030E77E /* Steps4ImpactSpec.swift */; };
 		2611A5052335FF290030E77E /* AKFCausesService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2611A5012335FF290030E77E /* AKFCausesService.swift */; };
-		2611A5062335FF290030E77E /* Facebook.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2611A5022335FF290030E77E /* Facebook.swift */; };
```

Comment:
Shouldn't this happen after the references are removed?  This commit wont build.

Reply:
good call - I'll reorder the commits

Reply:
not sure why GitHub isn't reflecting the new order on the PR but I moved the `delete Facebook services` commit on top of the changes. 
<img width="485" alt="Screen Shot 2020-06-28 at 6 25 22 PM" src="https://user-images.githubusercontent.com/1434903/85960960-ccb14e80-b96c-11ea-9aac-2fdba2da9770.png">